client/src/assets/scss/app.scss

```diff
@@ -0,0 +1,38 @@
+@import "./foundation/settings";
+@import "~foundation-sites/scss/foundation";
+@import "./foundation/variables"; 
+@include foundation-everything;
+
+body, html {
+  height: 100%;
+}
+
+.bg-img { 
+  background: linear-gradient(rgba(0, 0, -0.5, -0.5), 
+    rgba(0, 0, 0, 1)), 
+    url(../images/trees-fog-sharp.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  min-height: 100%;
+  min-width: 1024px;
+  width: 100%;
+  height: auto;
```

Comment:
You may want to inspect this class in the browser, these 4 properties for height/ min-height/ etc might be overwriting each other

Reply:
Right! I'm keeping a pair of "width: 100%; min-height: 100%" which seems to be working well. Thank you!